### PR TITLE
data: distributethe6ix

### PR DESCRIPTION
**Pull request by distributethe6ix**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
I hope I did this right (crosses fingers) 

:)

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2872"></a>

### CODE REVIEW

File: data/marinowijay.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      {
+        "name": "Tech Blogs",
+        "url": "https://marinow.hashnode.dev/",
+        "icon": "SiHashnode"
+      },
+      {
+        "name": "Youtube",
+        "url": "https://www.youtube.com/@marinowijay/streams",
+        "icon": "FaYouTube"
+      }
```
Comment (Aadarsh805) [inline data/marinowijay.json:43]: there is a comma missing at line 43

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/marinowijay.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+    "name": "Marino Wijay",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**DevRel Dojo** | Platform Advocate @ Solo.io | DevOps | Kubernetes | Service Mesh | Cloud-Native Open Source",
+    "tags": ["DevRel" "Cloud-Native Open Source", "DevOps", "Kubernetes", "Service Mesh", "Networking"],
```
Comment (Aadarsh805) [inline data/marinowijay.json:6]: comma missing here after `DevRel` in tags

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/marinowijay.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      },
+      {
+        "name": "Tech Blogs",
+        "url": "https://marinow.hashnode.dev/",
+        "icon": "SiHashnode"
+      },
+      {
+        "name": "Youtube",
+        "url": "https://www.youtube.com/@marinowijay/streams",
+        "icon": "FaYouTube"
```
Comment (Aadarsh805) [inline data/marinowijay.json:42]: typo in the youtube icon, it should be `FaYoutube`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/marinowijay.json (hunk 1 above)
Comment (distributethe6ix) [reply data/marinowijay.json]: Thank you! Fixed

File: data/marinowijay.json (hunk 2 above)
Comment (distributethe6ix) [reply data/marinowijay.json]: Thank you! Fixed

File: data/marinowijay.json (hunk 3 above)
Comment (distributethe6ix) [reply data/marinowijay.json]: Thank you! Fixed

**Review by distributethe6ix — COMMENTED** [review]
I have updated the fix with the change to correct the code with the missing commas, and icon labels.

**Review by Aadarsh805 — APPROVED** [review]
Alright, everything's great 👍

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, Marino.

Please fix your JSON profile file naming. It should be `{your GitHub username}.json` -> `
distributethe6ix.json`. Otherwise, features will not be functional.

If you need any help feel free to ask.

**Comment by distributethe6ix** [comment]
Hey @Pradumnasaraf updated the file name! Thanks for pointing that out!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**